### DISCUSSION TITLE
Misleading PR preview text

### DISCUSSION THREAD

**Discussion by itsmingjie** [community/community#70787] [Mobile]
A bit of an annoyance on the mobile app. Notifications sometimes show up with preview text "... requested your review..." but in fact it was another user's review that was requested. 

![image](https://github.com/community/community/assets/13333578/1d3f7f7a-82d7-4adb-a2e8-b9b1c81db38d)
![image](https://github.com/community/community/assets/13333578/e12f3841-c94f-4775-a4b9-a32decc51850)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.135.0 (112648507)
Device: iPhone16,1 (iOS 17.0.3)
</details>

**Comment by RobinSattler** [comment]
I'm encountering the same issue in a lot (possibly every) PR notification I receive. Are there any plans to fix this problem?
System info: Samsung Galaxy A23, Android 14, One UI 6.1

Btw. this is a duplicate of #36716 which sadly means that this issue exists for more than 2 years.
